.github/workflows/build-package-docs.yaml

```diff
@@ -0,0 +1,95 @@
+name: Ansible package docs build
+on:
+  workflow_dispatch:
+    inputs:
+      github_fork:
```

Comment:
It is customary to use `kebab-style` inputs in the GHA ecosystem, not `snake_case`. This also allows to visually distinguish between vars/identifiers when more languages are mixed up.